src/applications/vre/25-8832/config/form.js

```diff
@@ -0,0 +1,116 @@
+import fullSchema from 'vets-json-schema/dist/28-8832-schema.json';
```

Comment:
Probably won't need to update `vets-json-schema` any time soon

Reply:
Can you expand what you mean by this?